### PR TITLE
Jenkins Testing

### PR DESCRIPTION
**Pull request by jcpowermac**
WIP flag is to modify to the correct organization and repo.

This is the first pr to add testing to playbook2image.  Currently it will only build
the image in OpenShift.

This has been written to be used ephemerally:
- A seed job has been added to the S2I that will create the organization folder job.
- OpenShift Pipeline will create the Jenkins credential and start the seed job.
- The organization folder with github integration will use the Jenkinsfile at the root of the project to test pull requests.

### CODE REVIEW

**Comment by jcpowermac** [comment]
@codificat and @aweiteka there is also a global library that I put into another repo to make this work.  Not sure if I should just include it in this PR (and repo) or keep it separate.  What do you guys think?
https://github.com/jcpowermac/playbook2image-pipeline-library
It would just be the contents below `src/`

File: jenkins/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#### Using Jenkins to test playbook2image
+
+
+```
+oc process -f jenkins-ephemeral-template.yaml MEMORY_LIMIT=2Gi | oc create -f -
```
Comment (aweiteka) [inline jenkins/README.md]: Do we need a custom Jenkins template? Could we use the stock s2i jenkins template with our plugins.txt?

**Comment by aweiteka** [comment]
> a global library that I put into another repo to make this work.

Given the small size of the util I'd rather include it here for now unless you're using it somewhere else. We'll potentially have a lot of these so we need to find a common place. Also, others have developed libraries that we should look at. For example, https://github.com/fabric8io/fabric8-pipeline-library. I don't know what practice or patterns are emerging for sharing these things.

**Comment by jcpowermac** [comment]
@aweiteka and @codificat this pr is ready to go.  

@aweiteka I tested and made the modifications you suggested.  I also moved the library into this project but I agree we probably should think what to do with these pipeline libraries long term.

**Review by aweiteka — APPROVED** [review]

**Pull request merged**